content/en/docs/how-tos/notification.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Get notifications about a job"
```

Comment:
Please use title case like we do in other files. 

I also think we should mention Slack in the title, and respect the fact that we currently only support having this field on periodics (for generated jobs at least, generator does not respect that field on other jobs).

So maybe "Setup Slack Alerts for Periodic Job Results"?

Reply:
Updated.

Now i see we have this only for periodics:
https://github.com/openshift/ci-tools/blob/5065f7bbbfde5d61484be8649f17b4228a7228ac/pkg/jobconfig/files.go#L448

For all non-generated jobs with any type, we can define `reporter_config` right?